tests/behat/multilang_mastertemplate.feature

```diff
@@ -5,7 +5,8 @@ Feature: verify multilang in ATTLS (20 item version) mastertemplate
   I display a mastertemplate                                                  // The feature we want
 
   Background:
-    Given the following "courses" exist:
+    Given remote langimport tests are enabled
```

Comment:
You can configure (enable) it for travis:

https://github.com/moodlerooms/moodle-plugin-ci/blob/master/doc/AddExtraConfig.md